internal/api/delete_credentials_v1alpha1.go

```diff
@@ -15,34 +12,16 @@ func (s *server) DeleteCredentials(
 	ctx context.Context,
 	req *connect.Request[svcv1alpha1.DeleteCredentialsRequest],
 ) (*connect.Response[svcv1alpha1.DeleteCredentialsResponse], error) {
-	// Check if secret management is enabled
-	if !s.cfg.SecretManagementEnabled {
-		return nil, connect.NewError(connect.CodeUnimplemented, errSecretManagementDisabled)
-	}
-
-	project := req.Msg.GetProject()
-	if err := validateFieldNotEmpty("project", project); err != nil {
-		return nil, err
-	}
-
-	name := req.Msg.GetName()
-	if err := validateFieldNotEmpty("name", name); err != nil {
-		return nil, err
-	}
+	_, err := s.DeleteSecrets(ctx, &connect.Request[svcv1alpha1.DeleteSecretsRequest]{
+		Msg: &svcv1alpha1.DeleteSecretsRequest{
+			Project: req.Msg.GetProject(),
+			Name:    req.Msg.GetName(),
+		},
+	})
```

Comment:
Constructing a protobuf message to pass to another function seems a bit unconventional.

If you want to share Secret deletion logic between two endpoints, can it take project and name as arguments instead?